extra/generate-display-names/display-names

```diff
@@ -160,10 +160,10 @@
 * ../../tables/pt-pt-comp8.ctb                      Portuguese, computer, 8-dot                        Portuguese 8-dot computer braille
 * ../../tables/ro.tbl                               Romanian, computer                                 Romanian computer braille
 * ../../tables/ro-g0.utb                            Romanian, uncontracted                             Romanian braille
-* ../../tables/ru.ctb                               Russian, computer                                  Russian computer braille
+* ../../tables/ru.ctb                               Russian, computer, 8 dot                           Russian 8-dot computer braille
```

Comment:
Shouldn't this be `../../tables/ru-comp8.utb`

Reply:
Should be fixed.